uaclient/cli.py

```diff
@@ -1291,10 +1291,10 @@ def action_auto_attach(args, *, cfg):
     try:
         actions.auto_attach(cfg, instance)
     except exceptions.UrlError:
-        event.info(messages.ATTACH_FAILURE)
-        return 1
-    except exceptions.UserFacingError:
+        event.info(messages.ATTACH_FAILURE.msg)
         return 1
+    except exceptions.UserFacingError as e:
+        raise e
```

Comment:
Yeah, I noticed that the UserFacingError errors are not handled. @orndorffgrant @lucasmoura Do you foresee any implications of this?

Reply:
i.e. removing this line and leaving the main_error_handler decorator to handle it.

Reply:
Oh I didn't notice this was for auto-attach. Sorry we can't make this change.

The UserFacingError is already handled in actions.py. So if we don't catch it here, the msg will get printed more than once.
This should be refactored to use the main_error_handler instead of catching in actions.py, but that deserves its own PR.

Reply:
@CalvoM we still need to address this. I think if we delete actions.py line 43, then this change will be okay

Reply:
@orndorffgrant I had not seen this comment before. 

Yeah makes sense, lemme delete the extra logging line.